### PR TITLE
[build.pl] Add -testName and -fixture options

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

File: build.pl [hunk 1]
```diff
@@ -39,6 +39,9 @@
  verify    - rebuild the source with the binaries generated
              by the xbuild.
  tests     - run the tests.
+ runTestName - name of the nunit test to run (used with -tests)
+ fixture   - name of the nunit text fixture to be run (used
```
Comment (akoeplinger) [inline build.pl]: ~~text~~ test

File: build.pl [hunk 2]
```diff
@@ -39,6 +39,9 @@
  verify    - rebuild the source with the binaries generated
              by the xbuild.
  tests     - run the tests.
+ runTestName - name of the nunit test to run (used with -tests)
```
Comment (akoeplinger) [inline build.pl]: I think I'd prefer just `testname`.

You might also want to mention this needs to be the full name of the method incl. namespace

Comment (radical) [inline build.pl]: Sounds good. Updating the PR.

**Comment by AndyGerlicher** [comment]
LGTM

**Comment by radical** [comment]
Updated to fix merge conflict caused by 34b8c52cd69e563ded5521b855b659250fc32e26 .

**Pull request merged**